src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+.block {
+  height: 100px;
+}
+
+.red {
```

Comment:
In this task ok, but for the future, `green, red, blue ...` bad name for the classes.